src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> It would seem that even some basic unit tests would prevent those kinds of issues though.

I have same train of thought, i'm not sure such complication and size increase worth it